build-system/release-workflows/cut-nightly.js

```diff
@@ -33,12 +33,10 @@ async function getCommit(octokit) {
   );
 
   for (const {sha} of commits.data) {
-    const {'check_runs': checkRuns} = (
-      await octokit.rest.checks.listForRef({
-        ...params,
-        ref: sha,
-      })
-    ).data;
+    const checkRuns = await octokit.paginate(octokit.rest.checks.listForRef, {
+      ...params,
+      ref: sha,
+    });
```

Comment:
Don't actually know this API. I'm assuming the lack of `.data` at the end is intentional. Also that paginate will get _all_ the results instead of requiring multiple calls to it.

Reply:
I checked the API for the same reason 😛  looks ok https://octokit.github.io/rest.js/v18#pagination

Reply:
Thankfully our type checking stopped me before I could bork things up